src/vsr/superblock.zig

```diff
@@ -224,6 +219,7 @@ pub const SuperBlockHeader = extern struct {
             assert(state.checkpoint.manifest_oldest_checksum_padding == 0);
             assert(state.checkpoint.manifest_newest_checksum_padding == 0);
             assert(state.checkpoint.snapshots_block_checksum_padding == 0);
+            assert(state.checkpoint.free_set_last_block_checksum_padding == 0);
 
```

Comment:
Additionally can assert that `free_set_last_block_address == 0` implies `free_set_last_block_checksum == 0`.

Also `storage_size >= data_file_size_min`.